augly/image/functional.py

```diff
@@ -1765,6 +1765,55 @@ def shuffle_pixels(
     return imutils.ret_and_save_image(aug_image, output_path)
 
 
+def skew(
+    image: Union[str, Image.Image],
+    output_path: Optional[str] = None,
+    level: float = 0.5,
```

Comment:
Can we change this to skew_factor, which is more descriptive? Here and where it is referenced.
```suggestion
    skew_factor: float = 0.5,
```